src/main/kotlin/kweb/plugins/specificKeyUp/SpecificKeyUp.kt

```diff
@@ -1,7 +1,19 @@
-package kweb.html.events
+package kweb.plugins.specificKeyUp
 
 import kweb.Element
 import kweb.InputElement
+import kweb.html.events.KeyboardEvent
+import kweb.html.events.OnImmediateReceiver
+import kweb.html.events.OnReceiver
+import kweb.plugins.KwebPlugin
+
+/**
+ * @author sanity
+ *
+ * This plugin allows registering a listener on a specific key to avoid sending all `keyup` events
+ * to the backend.
+ */
+class SpecificKeyUpPlugin : KwebPlugin()
```

Comment:
Given that `SpecificKeyPlugin` doesn't require any configuration, could it be an `object` rather than a `class`?

Reply:
Sure, that should have been an object from the beginning :)